frontend/src/pages/SingleInstancePage.tsx

```diff
@@ -60,15 +65,79 @@ export default function SingleInstancesPage() {
           />
         </div>
       </ResizableContainer>
-      <Table
-        orientation="horizontal"
-        header={["Variable Name", "Variable Value", "Time"]}
-        content={
-          variables
-            ? variables.map(({ name, value, time }) => [name, value, time])
-            : []
-        }
-      />
+      <Tabs defaultValue={"variables"} className="bg-white">
+        <TabsList className="grid w-full grid-cols-2 rounded-lg border-2">
+          <Tab
+            value={"variables"}
+            className={`m-1 w-full rounded-md px-12 py-5 hover:bg-second-accent focus:bg-second-accent/100`}
+          >
+            Variables
+          </Tab>
+          <Tab
+            value={"jobs"}
+            className={`m-1 w-full rounded-md px-12 py-5 hover:bg-second-accent focus:bg-second-accent/100`}
+          >
+            Jobs
+          </Tab>
+        </TabsList>
+        <TabPanel value={"variables"}>
+          <VariablesTable variables={variables} />
+        </TabPanel>
+        <TabPanel value={"jobs"}>
+          <JobsTable jobs={jobs} />
+        </TabPanel>
+      </Tabs>
```

Comment:
These tab buttons are kind of huge, but at the same time hide to the background as it is not clear which is selected. I guess you can do styling later to improve on this.